Resources/Maps/Shuttles/DeltaV/listening_post.yml

```diff
@@ -3028,22 +2693,20 @@ entities:
       parent: 1
 - proto: ClosetWallBlack
   entities:
-  - uid: 405
+  - uid: 904
     components:
-    - type: MetaData
```

Comment:
Keep these named entities

Reply:
It shouldn't need that since it's an actual physical closet now with its own name and description-

> Added LPO Stamp Locker so now all newly added stamps are just a yaml fix

Reply:
Ok

Reply:
![image](https://github.com/user-attachments/assets/bd9abdd6-6483-4bca-a604-775dfa731406)

Reply:
nitpick but can we please rename the locker to either syndicate stamp locker or LPO stamp locker with caps